ArduPlane/Parameters.h

```diff
@@ -56,7 +56,7 @@ class Parameters {
         //
         k_param_auto_trim      = 10, // unused
         k_param_log_bitmask_old,  // unused
-        k_param_pitch_trim_cd,
+        k_param_pitch_trim,  // replaced by pitch_trim
```

Comment:
It doesn't need the comment. This is left over from when I thought it needed to be a new variable, nothing is being "replaced".

Reply:
removed